packages/hub/prisma/schema.prisma

```diff
@@ -235,6 +235,8 @@ model ModelExport {
   modelRevisionId String
   variableName    String
   title           String?
+  docstring       String?
```

Comment:
When we don't expect to make a distinction between null and an empty string (which I think applies here), in my experience it's better to make it non-nullable. Nulls can cause subtle bugs where you accidentally check for one empty value variation but not for another.

So I'd recommend `String @default("")` for `title` and for `docstring`.

Reply:
This also applies to `ModelExport` type in components and to the GraphQL schema (on output fields; on inputs, leaving them nullable is fine).

Reply:
Do you think I should make a new migration for this, or reset the db?

Reply:
(Never mind, done)

Reply:
Doesn't matter, but I would choose a reset.

Each migration has a minor associated cost; I can [squash migrations](https://www.prisma.io/docs/guides/migrate/developing-with-prisma-migrate/squashing-migrations) if we accumulate too many of them, but I'd prefer not to do it too often. So it's better if we have no more than one migration per PR.